tests/conftest.py

```diff
@@ -40,11 +32,18 @@ def govuk_alerts():
     ctx.push()
 
     yield app
-
     ctx.pop()
 
 
-def render_template(env, template_path, template_vars={}):
-    template = env.get_template(template_path)
-    content = template.render(template_vars)
-    return BeautifulSoup(content, 'html.parser')
+@pytest.fixture()
+def client_get(govuk_alerts, mocker):
+    mocker.patch('app.models.alerts.Alerts.load', return_value=Alerts([]))
```

Comment:
Optional - It might make it slightly more obvious what is going on and what is being mocked if this was in a different fixture, particularly since it gets overwritten in one of the tests.

This would mean that most tests would now need a fixture like `mock_no_alerts_returned` as well as the `client_get` fixture.

Reply:
Yeah, I was in two minds about this. I didn't seem worth cluttering most of the tests with the extra mock. 

A couple of other options:

- Add a comment above this line.
- Make it a fixture but require it here (still).

My thinking behind putting the mock here was based on providing a clean base for most tests to operate with. It's easier to do this in things like RSpec with a `before(:each)`. We could also do this:

```
@pytest.fixture(autouse=True)
def mock_no_alerts(): ...
```

But I'm a bit allergic to autouse fixtures because the tag is quite subtle.

Do you still lean towards a mock in each test?

Reply:
I also don't really like `autouse`, so let's avoid that. At the moment, there's only one test which overwrites the mock and returns some data instead. How about leaving things as they are for now, but potentially splitting this out into a separate mock for clarity if we get to a situation where there are more tests which need `'app.models.alerts.Alerts.load'` to actually return some alerts.